flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlMaterializedTableNodeToOperationConverterTest.java

```diff
@@ -1022,4 +1039,56 @@ private static List<Arguments> createOrAlter(final String operation) {
                                         "PK_user_id", List.of("user_id")),
                                 List.of())));
     }
+
+    private CatalogMaterializedTable.Builder getDefaultMaterializedTablebuilder() {
```

Comment:
maybe a one liner comment to define the "defaultMaterializedTable" meaning, could help to understand this method and tests better